sdk/core/azure-core/azure/core/tracing/decorator.py

```diff
@@ -61,6 +62,7 @@ def distributed_trace(  # pylint:disable=function-redefined
     """
     name_of_span = kwargs.pop("name_of_span", None)
     tracing_attributes = kwargs.pop("tracing_attributes", {})
+    kind = kwargs.pop("kind", SpanKind.INTERNAL)
```

Comment:
Why `internal`? We are about to make a network request, so I would have assumed that `client` is the right value.

Reply:
My understanding that it affects logical (client method call) spans and those should be `internal`, http spans should be `client`. The reason - AzMon (and presumably other tracing backends) needs a way to distinguish logical and physical calls to properly show application maps (outgoing edges and leaf Azure services nodes)

Reply:
@johanste indeed that's the decorator for the client method call, so following @lmolkova 's guidance I used internal :)